test_file

```diff
@@ -0,0 +1 @@
+1,2,3
```

Comment:
Do we want to commit this?

Reply:
No! that was bug in the tests that should now be fixed (it was creating a test file which I've now fully mocked)

Reply:
Can we delete this file from the commit then (if we haven't already)?

Reply:
Ooh okay -  can we remove it from the commit then?

Reply:
Yes pls do

Reply:
_[Marked as unresolved]_ Is it just me or is this still showing up in the repo?